spec/config/config_parser_spec.rb

```diff
@@ -364,24 +364,24 @@ module Cosmos
           expect(ConfigParser.handle_defined_constants("MIN", :UINT, val)).to eql 0
           expect(ConfigParser.handle_defined_constants("MAX", :UINT, val)).to eql (2**val - 1)
           # Signed
-          expect(ConfigParser.handle_defined_constants("MIN", :INT, val)).to eql -((2**val) / 2)
+          expect(ConfigParser.handle_defined_constants("MIN", :INT, val)).to eql (-(2**val) / 2)
           expect(ConfigParser.handle_defined_constants("MAX", :INT, val)).to eql ((2**val) / 2 - 1)
         end
         [8,16,32,64].each do |val|
           # Unsigned
           expect(ConfigParser.handle_defined_constants("MIN_UINT#{val}")).to eql 0
           expect(ConfigParser.handle_defined_constants("MAX_UINT#{val}")).to eql (2**val - 1)
           # Signed
-          expect(ConfigParser.handle_defined_constants("MIN_INT#{val}")).to eql -((2**val) / 2)
+          expect(ConfigParser.handle_defined_constants("MIN_INT#{val}")).to eql (-(2**val) / 2)
           expect(ConfigParser.handle_defined_constants("MAX_INT#{val}")).to eql ((2**val) / 2 - 1)
         end
         # Float
         expect(ConfigParser.handle_defined_constants("MIN_FLOAT32")).to be <= -3.4 * 10**38
         expect(ConfigParser.handle_defined_constants("MAX_FLOAT32")).to be >= 3.4 * 10**38
-        expect(ConfigParser.handle_defined_constants("MIN_FLOAT64")).to eql -Float::MAX
+        expect(ConfigParser.handle_defined_constants("MIN_FLOAT64")).to eql (-Float::MAX)
         expect(ConfigParser.handle_defined_constants("MAX_FLOAT64")).to eql Float::MAX
         expect(ConfigParser.handle_defined_constants("POS_INFINITY")).to eql Float::INFINITY
-        expect(ConfigParser.handle_defined_constants("NEG_INFINITY")).to eql -Float::INFINITY
+        expect(ConfigParser.handle_defined_constants("NEG_INFINITY")).to eql (-Float::INFINITY)
```

Comment:
Having a negative in front of a variable generates a warning because maybe you meant subtraction (I guess that's the rationale). Anyway I started doing this but then gave up because it's annoying and looks weird to have all these values in parens. Plus we only do this in specs.